README.md

```diff
@@ -7,180 +7,211 @@ gollum -- A wiki built on top of Git
+| Windows          | Ruby 1.9.3+    | none               | no        |
+| Windows          | [JRuby](https://github.com/jruby/jruby) (1.9.3+ compatible) | [RJGit](https://github.com/repotag/rjgit) | almost<sup>1</sup> |
 
-## SECURITY
+**Notes:**
 
-Don't enable `--custom-css`, `--custom-js` or `--mathjax-config` unless you trust every user who has the ability to edit the wiki.
-A better solution with more security is being tracked in [#665](https://github.com/gollum/gollum/issues/665).
+1. There are still some bugs and this setup is not ready for production yet. You can track the progress at #1044.
 
 ## INSTALLATION
 
-The best way to install Gollum is with RubyGems:
-
-```bash
-$ [sudo] gem install gollum
-```
-You may first need to install some additional [development packages](https://github.com/gollum/gollum/wiki/Installation) for your OS.
-
-If you're installing from source, you can use [Bundler][bundler] to pick up all the
-gems:
-
-```bash
-$ bundle install
-```
-
-In order to use the various formats that Gollum supports, you will need to
-separately install the necessary dependencies for each format. You only need
-to install the dependencies for the formats that you plan to use.
-
-* [AsciiDoc](http://asciidoctor.org) -- `gem install asciidoctor`
-* [Creole](http://www.wikicreole.org/) -- `gem install creole`
-* [Markdown](http://daringfireball.net/projects/markdown/) -- `gem install redcarpet`
-* [GitHub Flavored Markdown](https://help.github.com/articles/github-flavored-markdown) -- `gem install github-markdown`
-* [Org](http://orgmode.org/) -- `gem install org-ruby`
-* [Pod](http://search.cpan.org/dist/perl/pod/perlpod.pod) -- `Pod::Simple::HTML` comes with Perl >= 5.10. Lower versions should install Pod::Simple from CPAN.
+Varies depending on operating system, package manager and Ruby installation. Generally, you should first install Ruby and then Gollum.
+
+1. Ruby is best installed either via [RVM](https://rvm.io/) or a package manager of choice.
+2. Gollum is best installed via RubyGems:  
+	```
+	[sudo] gem install gollum
+	```
+
+Installation examples for individual systems can be seen [here](https://github.com/gollum/gollum/wiki/Installation).
+
+**Notes:**  
+* Whichever Ruby implementation you're using, Gollum ships with the appropriate default git adapter. So the above installation procedure is common for both MRI and JRuby.
+* If you're installing from source:
+	* Optionally uninstall any previous versions of Gollum:  
+		```
+		[sudo] gem uninstall -aIx gollum
+		```
+	* Install [Bundler](http://bundler.io/).
+	* Navigate to the cloned source of Gollum.
+	* Install dependencies:  
+		```
+		[sudo] bundle install
+		```
+	* And finally:  
+		```
+		rake build
+		[sudo] gem install --no-ri --no-rdoc pkg/gollum*.gem
+		```
+
+### Markups
+
+Gollum presently ships with support for the following markups:
+* [Markdown (Kramdown flavour)](http://kramdown.gettalong.org/quickref.html)
```

Comment:
Ah, I see. Didn't read kramdown's documentation since my interent was directed at the quick reference :).

Shall I just simply replace the "Kramdown flavour" with "Markdown" then?